src/openMVG/matching_image_collection/gpu/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(openMVG_gpu_matching_image_collection)
+
+if (OpenMVG_USE_CUDA)
```

Comment:
yep, my bad, I though we was in a c++ related file, but here it's a cmake file. Sorry for the confusion.